internal/commands/pull/command.go

```diff
@@ -138,6 +165,14 @@ func (cmd *Command) Handler(profile *user.Profile, ui terminal.UI, clients cli.C
 		ui.Print(terminal.NewDebugLog("Fetched hosting assets"))
 	}
 
+	for templateID, templateZipPkg := range clientZipPkgs {
+		if err := local.WriteZip(pathFrontend, templateZipPkg); err != nil {
```

Comment:
I'd rather see us do this:

```go
if err := local.WriteZip(pathFrontend, templateZipPkg); err != nil {
  // note also the lack of starting capital letter, single quotes around the template id, and `%s` for the error string
  return fmt.Errorf("Unable to save template '%s' to disk: %s", templateID, err)
} else {
  // then also add a TODO here...
  // TODO(REALMC-9452): defer printing the successfully saved templates until after the `Successfully pulled app down' log
  // we should be able to print the list of saved templates in one log message/list, rather than each on their own new line
  ui.Print(terminal.NewTextLog("Saved template %s to disk", templateID))
}
```